Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM ruby:2.4-stretch
+FROM ruby:2.6-alpine
+RUN apk update \
+    && apk upgrade \
+    && apk add --update --no-cache g++ musl-dev make libstdc++ yarn
```

Comment:
Maybe the package list it is easier to read & maintain in multiline format, something like:

```
pk add --update --no-cache \
 g++ \
 musl-dev \ 
make \
libstdc++ \ 
yarn
```